pages/management/ManagementHandler.php

```diff
@@ -16,20 +16,15 @@
 namespace PKP\pages\management;
 
 use APP\components\forms\context\DoiSetupSettingsForm;
-use APP\components\forms\context\MetadataSettingsForm;
-use APP\components\forms\context\ReviewGuidanceForm;
 use APP\core\Application;
 use APP\core\Request;
 use APP\facades\Repo;
 use APP\file\PublicFileManager;
 use APP\handler\Handler;
 use APP\template\TemplateManager;
-use PKP\components\forms\context\PKPDisableSubmissionsForm;
 use PKP\components\forms\context\PKPEmailSetupForm;
 use PKP\components\forms\context\PKPInformationForm;
 use PKP\components\forms\context\PKPNotifyUsersForm;
-use PKP\components\forms\context\PKPReviewSetupForm;
-use PKP\components\forms\context\PKPSubmissionsNotificationsForm;
```

Comment:
Updated `PKPReviewSetupForm` but it looks like `PKPSubmissionsNotificationsForm` doesn't exist. I'll double check that it wasn't added since I last rebased.